content/developer/misc/api/iap.rst

```diff
@@ -545,6 +549,8 @@ Authorize
                             charges on their account
     :param str dbuuid: optional, allows the user to benefit from trial
                        credits if his database is eligible (see :ref:`Service registration <register-service>`)
+    :param int ttl: optional, transaction time to live in hours. If the credit are not captured when the transaction expires,
+                    the transaction is cancelled
```

Comment:
optional, transaction time to live in hours. If the credit are not captured when the transaction expires, the transaction is cancelled. Default value: 4320 (?)